GameData/RP-0/Contracts/Space Stations/Improved Space Station.cfg

```diff
@@ -0,0 +1,606 @@
+			disableOnStateChange = false
+			title = Remain in a Stable Orbit
+			completeInSequence = true
+		}	
+		PARAMETER
+		{
+			name = Duration
+			type = Duration
+			duration =  60d
+			preWaitText = Transfer to the station in the specified orbit.
+			waitingText = Orbiting...
+			completionText = Stay completed, you may return home now.
+			completeInSequence = true
+		}
+		PARAMETER
+		{
+			name = CollectSci
+			type = CollectScience
+			situation = InSpaceLow
+			recoveryMethod = RecoverOrTransmit
+		}
+		disableOnStateChange = true
+	}
+	PARAMETER
+	{
+		name = vesselGroupCrewReturn
+		type = VesselParameterGroup
+		title = Return the crew home
+		vessel = crewCapsule
+
+		completeInSequence = true
+
+		PARAMETER 
+		{
+			name = Crewmembers
+			type = HasCrew
+			minCrew = 3
+			title = Have at least 3 crewmembers on board
+			hideChildren = true
+		}
+		PARAMETER
+		{
+			name = ReturnHome
+			type = ReturnHome
+			targetBody = HomeWorld()
+			title = Return Home Safely
+			hideChildren = true
+			completeInSequence = true
+		}
+	}
+	
+	BEHAVIOUR
+	{
+		name = IncrementCompletions
+		type = Expression
+		
+		CONTRACT_COMPLETED_SUCCESS
+        {
+            first_ISScrew_Count = $first_ISScrew_Count + 1
+        }
+	}
+}
+
+CONTRACT_TYPE
+{
+	name = second_ISScrew
+	title = Crew your Improved Space Station for 90 days and collect science
+	group = SpaceStations
+	agent = Stations
+	
+	description = You have succeeded in crewing your station for 2 months. Now it's time for you to crew it for longer. Crew your station for 90 days and then return safely. This mission can be completed 2 times. Afterwards, you are free to deorbit the station. &br;<b><color=red>Do NOT deorbit the station while you are completing this contract. Otherwise, this contract will fail and you will be forced to redo the Improved Space Station contract</color></b> &br;&br;<b>Number of Contracts Completed: $second_ISScrew_Count</b> 
+	genericDescription = Crew your station for 90 days and collect science, then return safely.
+	
+	synopsis = Crew your station for 3 months.
+	
+	completedMessage = Congratulations! You have lived on a space station for 90 days. This has given us good data on long term crew survivability in space.
+	
+	sortKey = 1512
+	
+	cancellable = true
+	declinable = false
+	autoAccept = false
+	minExpiry = 0
+	maxExpiry = 0
+	maxCompletions = 2
+	maxSimultaneous = 1
+	deadline = 365 * RP1DeadlineMult() // 1 year
+	
+	targetBody = HomeWorld()
+	
+	// ************ REWARDS ************
+	
+	prestige = Significant // 1.25x
+	advanceFunds = 0
+	rewardScience = 0
+	rewardReputation = 50
+	rewardFunds = 0
+	failureReputation = @rewardReputation
+	failureFunds = 0
+	
+	// ************ REQUIREMENTS ************
+	
+	REQUIREMENT
+	{
+		name = ProgramActive
+		type = ProgramActive
+		program = EarthSpaceStation
+	}
+	REQUIREMENT
+	{
+		name = CompleteContract
+		type = CompleteContract
+		contractType = first_ISScrew
+		minCount = 2
+	}
+	RREQUIREMENT
```

Comment:
typo

Reply:
Also, in order for these contracts to show up, [this contract group](https://github.com/KSP-RO/RP-0/blob/NotBudgets/GameData/RP-0/Contracts/Groups.cfg#L124) needs to be renamed to SpaceStations

Reply:
Thanks for pointing out the typo, will get to fixing it.

Wasn’t the contract group and program renamed from SpaceStations to EarthSpaceStation, presumably with a Lunar, Martian, etc programs planned in the future? I noticed that and decided to go along with it.

Reply:
Sure, but your changes in this PR renamed the group of all contracts back to SpaceStations. So either rename them back to EarthSpaceStation or rename the entire group to SpaceStations, preferably the former :)